api/pkg/app/app.go

```diff
@@ -410,8 +410,8 @@ func initGh() (*GHConfig, error) {
 		}
 
 		ghe.IsGhe = true
-		ghe.ApiUrl = fmt.Sprintf("%s://api.%s", parsedUrl.Scheme, parsedUrl.Host)        // https://api.myghe.com
-		ghe.UploadUrl = fmt.Sprintf("%s://uploads.%s", parsedUrl.Scheme, parsedUrl.Host) // https://uploads.myghe.com
+		ghe.ApiUrl = fmt.Sprintf("%s://%s", parsedUrl.Scheme, parsedUrl.Host)        // https://myghe.com
+		ghe.UploadUrl = fmt.Sprintf("%s://%s", parsedUrl.Scheme, parsedUrl.Host) // https://myghe.com
```

Comment:
can we remove this since we are already having this url in `ghe.Url`?

Reply:
Done, PTAL